### ISSUE TITLE
arm servers support

### ISSUE THREAD

**Issue by lironsher** [getredash/setup#77]
i'm try to install on ubuntu 22.04 on OCI arm cpu. but it fails to tun docker compose with the error:

```
*********************
** Starting Redash **
*********************
** Initialising Redash database **
[+] Creating 2/0
 ✔ Container redash-redis-1     Running                                                                                                 0.0s 
 ✔ Container redash-postgres-1  Running                                                                                                 0.0s 
exec /app/bin/docker-entrypoint: exec format error
```

I've tested the on ubuntu 22.04 on OCI AMD cpu and i do get the url for the website

can someone advise?

**Comment by lironsher** [comment]
Soultion

Install QEMU: QEMU allows you to emulate one CPU architecture on another, such as x86 on ARM.

Install the necessary packages to support QEMU emulation in Docker:

$ sudo apt-get install qemu qemu-user-static

install the support:

$ docker run --rm --privileged tonistiigi/binfmt --install all

**Comment by justinclift** [comment]
As a data point, we (in theory) support ARM64.  Well, we went through the process of getting it working for developers using Apple Silicon mac's somewhat recently.

I'm not if sure the "ARM" you're talking about is ARM64 or not though. ?

**Comment by lironsher** [comment]
OCI running Ampere ARM CPU - https://amperecomputing.com/

I've managed to install it after adding the QEMU support,

But when I try to access to website on port 5000 - it's hangs

meanwhile i'm running on AMD and it's working OK. (SAME OS)

**Comment by unfixa1** [comment]
Docker restarts the image or it is recommended that you use Servbay to replace Docker to deploy the development environment. [Servbay ](servbay.com)supports arm architecture.

**Comment by unfixa1** [comment]
Docker image does not support ARM architecture
It is recommended that you use [Servbay](servbay.com) to replace docker to deploy the development environment. Servbay has fewer bugs and supports the arm architecture.

**Comment by lucydodo** [comment]
Yes, we do support ARM64 architecture, but currently only in 'preview' tags.
When running the installation script, you will need to provide the `--preview` additional command arguments.
@lironsher Can you run it using an ARM64 build rather than QEMU emulation and let me know?

**Comment by lironsher** [comment]
Where should I add the tags?

**Comment by lucydodo** [comment]
Here are the installation instructions for each method:
1. Docker: `docker pull redash/preview`
2. Our Setup Script: `./setup.sh --preview`
